building-applications/instances.md

```diff
@@ -33,6 +33,27 @@ instance:
     type: "kubernetes"
 ```
 
+### Globals
+
+Within instance.yaml, use "globals" to define values for parameters across your application.
+
+For example, this instance defines topicName as a global parameter with the value "input-topic":
+```yaml
+instance:
+  globals:
+    topicName: "input-topic"
```

Comment:
Here you can refer to ENV variables ${VARIABLE:-default value} like in secrets.yaml